integrations/discord/websockets/message.go

```diff
@@ -0,0 +1,35 @@
+package websockets
+
+import (
+	"context"
+
+	"github.com/bwmarrin/discordgo"
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/integrations/discord/internal/vars"
+)
+
+func (h *handler) handleMessage(_ *discordgo.Session, m any, eventType string) {
```

Comment:
1. No need to pass the session
2. `m` is short which is good, but when its type doesn't explain what it is, it's better to use a slightly longer name
```suggestion
func (h *handler) handleMessage(msg any, eventType string) {
```